spec/restful_resource_bugsnag_spec.rb

```diff
@@ -66,12 +66,12 @@
   describe 'when a notification is sent for an OtherHttpError error' do
     let(:response) do
       {
-        :status => 503,
+        :status => 500,
```

Comment:
yup I think so. Any response that is not a 404, 422 or 503 is an OtherHttpError:
https://github.com/carwow/restful_resource/blob/master/lib/restful_resource/http_client.rb#L86-L93